gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -255,12 +255,17 @@ user:
     email:
       #subject: Please reset your password
     token:
-      #expire-after: 86400 # the end of validity of the token in seconds for reset password email (default 86400 = a day)
+      #expire-after: 300 # the end of validity of the token in seconds for reset password email (default 300 = 5 minutes)
   blockedAccount:
       email:
       #subject: Account has been locked
       token:
       #expire-after: 86400 # the end of validity of the token in seconds for blocked account email (default 86400 = a day)
```

Comment:
Why not 300 seconds as well for the blocked account one ?

Reply:
to allow the user to restore his account if he wasn't responsible of the blocking action